src/pages/Content/index.js

```diff
@@ -6,6 +12,8 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
     case 'startRecording':
       console.log('Should start recording here');
       sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
+      // attach global event listeners
+      attachGlobalEventListeners();
```

Comment:
Seems like the correct place.
Ps maybe also take the `detchGlobalEventListeners` because you're already warm with it? or help one of the others with guidance?

Reply:
I'm super busy until March 1st, but I'm available to help right after, I've cleared an entire week to work on the project.